src/nav/components.yml

```diff
@@ -80,9 +80,9 @@
       id: https://zendeskgarden.github.io/react-components/modals
     - title: Notification
       items:
-        - id: https://zendeskgarden.github.io/react-components/notifications
+        - id: /components/alert
           title: Alert
-        - id: https://zendeskgarden.github.io/react-components/notifications
+        - id: /components/notification
```

Comment:
There are multiple types of notifications, should this be plural?

Reply:
Yes, this should be `Notifications`